### ISSUE TITLE
Source of the CO2 restart files?

### ISSUE THREAD

**Issue by garvitagarwal290** [geoschem/geos-chem#2162]
### Name and Institution (Required)

Name: Garvit Agarwal
Institution: TCS Research, Pune, India

### Confirm you have reviewed the following documentation

- [ ] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/SUPPORT.html)
- [ ] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [ ] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [ ] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question

How were the initial concentrations of CO2 obtained in restart files like 'initial_GEOSChem_rst.2x25_CO2.nc' at http://geoschemdata.wustl.edu/ExtData/GEOSCHEM_RESTARTS/v2018-11/? In the README it says, "These files were produced from the BPCH restart files used in GEOS-Chem v10-01 and earlier" but I dont know what the BPCH restart files are and neither is the GEOS-Chem v10-01 folder available. Could you please explain simply how the initial concentrations (in the above file) were obtained and what year they belong to? Thank you.

**Comment by yantosca** [comment]
@garvitagarwal290: Restart files do not contain observational data, but is just the state of the model atmosphere at the end of a previous GEOS-Chem simulation. 

We do have some documentation online about restart files on ReadTheDocs:
- https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/restart-files.html

**Comment by zion0210** [comment]
Hello!

I want to run simulations of CO2 starting from 2010.
But initial CO2 conditions is available from 2019-01-01.

According to the documentation 
(https://geos-chem.readthedocs.io/en/stable/gcclassic-user-guide/restart-files.html), 
I should take data for 2019-01-01 and make "spin up".
Namely, I run GH starting from 2010-01-01 and ending in 2011-01-01  and used initial CO2 conditions from 2019-01-01.
After, I took result data for 2011-01-01 and run GH beginning in 2010. I did this 10 times. 
Each subsequent result differed from the previous one by +15 ppm.

I assumed that this difference should decrease at each subsequent stage of “spin up”.

It is normal? Maybe, I made something wrong?

Also, I found result for monthly full chemical benchmark 
(https://ftp.as.harvard.edu/gcgrid/geos-chem/10yr_benchmarks/14.0.0/GCClassic/Restarts/)
But the data of CO2 array of several orders differs from the data that is distributed ont the Wustl or Rochester repositories. 

Best regards,
zion0210
